<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QMainWindow" name="Demo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Демонстрация</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="7,0,0,5,1" columnstretch="10,0">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_input">
      <item>
       <widget class="QLineEdit" name="input1">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Введите первое число</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input2">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Введите второе число</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="p_semi">
       <attribute name="title">
        <string>P-полупроводник</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayout_pSemi"/>
      </widget>
      <widget class="QWidget" name="n_semi">
       <attribute name="title">
        <string>N-полупроводник</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayout_nSemi"/>
      </widget>
      <widget class="QWidget" name="diod">
       <attribute name="title">
        <string>P-n переход</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayoutDiod"/>
      </widget>
      <widget class="QWidget" name="transistor">
       <attribute name="title">
        <string>Транзистор</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayout_transistor"/>
      </widget>
      <widget class="QWidget" name="halfbitsum">
       <attribute name="title">
        <string>Полубитный сумматор</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayout_halfbitsum"/>
      </widget>
      <widget class="QWidget" name="twobitsum">
       <attribute name="title">
        <string>2-битный сумматор</string>
       </attribute>
       <layout class="QGridLayout" name="graphicLayout_twobitsum"/>
      </widget>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_changePolarity">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Изменить полярность подключения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_toMenu">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>В главное меню</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
